art/art_dic.py

```diff
@@ -472,6 +472,14 @@
     "playing cards hearts": "[♥]]]",
     "playing cards diamonds": "[♦]]]",
     "playing cards spades": "[♠]]]",
-    "playing cards clubs": "[♣]]]"
+    "playing cards clubs": "[♣]]]",
+    "black spade": "♠",
+    "black heart": "♥",
+    "black diamond": "♦",
+    "black club": "♣",
+    "white spade": "♤",
+    "white heart": "♡",
+    "white diamond": "♢",
+    "white club": "♧"
```

Comment:
Can we consider a better name for these sets?
black sets are colorful in some operating systems and i think `bold heart` and so on and `regular hear` and ... or any other suitable name.

Reply:
Done! https://github.com/sepandhaghighi/art/pull/92/commits/7602192355dd1ad43d7b32daa72b9fed757733b3